services/py-api/src/server/exception.py

```diff
@@ -7,3 +9,8 @@ class DuplicateEmailError(DuplicateKeyError):
 
 class DuplicateTeamNameError(DuplicateKeyError):
     pass
+
+class CapacityExceededError(HTTPException):
```

Comment:
I understand why you added this, but I am not sure we need it to be Inheriting from HTTPException, Exception is enough.

```python
class CapacityExceededError(Exception):
         pass
```